src/pages/projects/native.astro

```diff
@@ -42,25 +42,26 @@ import Layout from "../../layouts/Layout.astro";
     <div class="container">
       <h1 class="display-4 mb-3">MapLibre Native</h1>
       <p class="lead mb-4 opacity-75">
-        The fully open-source, high-fidelity mapping engine that delivers
-        exceptional performance across all platforms
+        The battle-tested, open-source mapping engine built for exceptional
```

Comment:
given that that one military company is using us, battle tested might not even be a marketing ploy